style.css

```diff
@@ -0,0 +1,686 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,400;1,700&family=Poppins&family=Roboto:wght@400;500;700;900&display=swap");
+
+:root {
+  --ff-primary: "Lato", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html,
+body {
+  scroll-behavior: smooth;
+}
+
+body {
+  color: #fff;
+  overflow-x: hidden;
+  font-family: "Lato", sans-serif;
+}
+
+.flex-nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.2rem;
+  height: 4rem;
+}
+
+.nav-list2 {
+  display: none;
+  flex-direction: column;
+  gap: 2rem;
+  background-color: #272a31;
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  padding: 1rem;
+  z-index: 1;
+}
+
+.clicklogo {
+  list-style: none;
+  display: none;
+}
+
+.clicklogo img {
+  width: 20%;
+}
+
+.prog img {
+  width: 20%;
+  height: 20%;
+}
+
+.chefs img {
+  width: 40%;
+  height: 40%;
+}
+
+.click,
+a {
+  list-style: none;
+  justify-content: space-between;
+  text-decoration: none;
+  color: #d3d3d3;
+  font-size: 3rem;
+}
+
+header {
+  background-image: url("./img/high10.jpg");
+  background-repeat: repeat;
+  background-size: cover;
+  width: 100%;
+  height: auto;
+}
```

Comment:
- [ ]  Your mobile background should be transparent or the same color/pattern as the top section background.